src/neuroglancer/segmentation_user_layer.ts

```diff
@@ -159,6 +163,17 @@ export class SegmentationUserLayer extends UserLayer {
         });
       }
     });
+
+    verifyObjectProperty(x, 'clipBounds', y => {
+      if (y === undefined) {
+        return;
+      }
+      let bounds = {
+        center: vec3.clone(y['center']),
```

Comment:
Perhaps use verifyObjectProperty(y, 'center', ...), and parseFiniteVec, etc. to more thoroughly check format